circleci/docker-publish

```diff
@@ -61,7 +71,7 @@ fi
 
 # dockerhub login.
 # `FROM image` in Dockerfile is going to pull from DockerHub
-docker login -u $DOCKER_USER -p $DOCKER_PASS --email="$DOCKER_EMAIL" || docker login -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
Nit. If `DOCKER_EMAIL` is no longer needed, we should removed it from `check_hub_vars`